engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableSliceTest.java

```diff
@@ -442,6 +443,28 @@ public void testHeadTailPct() {
                 TstUtils.testRefreshingTable(i(2).toTracking(), col("x", 1), col("y", 'a')));
         assertTableEquals(table.tailPct(0.1),
                 TstUtils.testRefreshingTable(i(6).toTracking(), col("x", 3), col("y", 'c')));
+
```

Comment:
I would add tests at 0,1 as well.  If we decide inclusive, they should pass and have either 0 rows or be the same size as the input table.  If exclusive, should result in exceptions.